### ISSUE TITLE
关于多分类问题

### ISSUE THREAD

**Issue by MarcHo1994** [zhixuhao/unet#18]
感谢你的分享，也是看了你的博客而来。
想请教一个关于多分类的问题：
这个UNet如何做到多分类呢？像FCN那样，应该怎么修改呢？
感谢。

**Comment by Codekiller-Li** [comment]
同样提问
        conv10 = Conv2D(3,(1,1),activation='softmax')(conv9)
        model = Model(inputs=inputs, outputs=conv10)
        model.compile(optimizer=Adam(lr=1e-4), loss='binary_crossentropy', metrics=['accuracy'])
我做打算做三分类图片，图片已经进行了one-hot编码,但model结果不对，请问如何能正确改成多分类呀

**Comment by flighted** [comment]
请问有解决方案了么？怎么做？

**Comment by mingzhaochina** [comment]
label只是一通道的，而图片是三通道的，做多分类的话，是不是每个类别都要单独做一个标签呢？否则没法计算交叉商loss啊，要求是label和数据是一样的shape

**Comment by Codekiller-Li** [comment]
可以看我的github，我用这个改了一个做三分类的

**Comment by leo-sy** [comment]
将最后一层的激励函数改为softmax，损失函数改为sparse_categorical_crossentropy，在对图像按照类别进行预处理，就可以进行多分类。

**Comment by Monologuethl** [comment]
请问”图像按照类别进行预处理，就可以进行多分类“，具体是怎么操作呢

**Comment by mqray** [comment]
I have found some useful solution, you can refer `https://github.com/wuyang0329/unet`

**Comment by Monologuethl** [comment]
thank you ,i have solved this porblem !

**Comment by dreamandgo-123** [comment]
> thank you ,i have solved this porblem !

你好，方便说一下是怎么做的嘛？
